src/index.ts

```diff
@@ -45,3 +49,20 @@ export const buildServicesFromConfig = async (): Promise<Services> => {
 
     return {ceramic}
 }
+
+export default class IntegrationTestEnvironment extends NodeEnvironment {
+    async setup() {
+        await super.setup();
+
+        const services = await buildServicesFromConfig();
+        this.global.services = services;
```

Comment:
Rather than adding this `services` object, maybe all the needed values could be set on the global object directly? For example: https://github.com/ceramicnetwork/jest-environment-ceramic/blob/master/index.js#L39-L42

This way they can be referenced to directly (https://github.com/ceramicstudio/js-idx/blob/master/test/integration.test.ts#L19 for example), no need for extra `beforeAll` logic as in https://github.com/3box/ceramic-integration-tests/pull/4/files#diff-9dfa8b87bfa3e1d314472cc8b40f0221dc5f4c84c61afaf2da82ed8f4c5de208R33

So here doing `this.global = await buildServicesFromConfig()` should do the trick.

Reply:
I was trying to think about the future where there were multiple services, and how it'd be nice to be able to unpack all the services on a single line, rather than having a bunch of lines to declare individual variables pulled from the global one at a time.  But I didn't know about the `declare global` block syntax, looks like that might handle that concern nicely.  I'm going to give it a try now

Reply:
Just trying to assign the entire `Services` object to `global` gave me an error:
```
src/index.ts:61:9 - error TS2740: Type 'Services' is missing the following properties from type 'Global': __coverage__, jasmine, fail, pending, and 75 more.

61         this.global = await buildServicesFromConfig();
           ~~~~~~~~~~~
```
So I just changed it so that `buildServicesFromConfig()` installs the individual services into the `global` directly